lib/s3_meta_sync/syncer.rb

```diff
@@ -16,7 +17,12 @@ class Syncer
     DEFAULT_REGION = 'us-east-1'
     STAGING_AREA_PREFIX = "s3ms_"
 
+    AWS_PUBLIC_ACCESS  = 'public-read'
+    AWS_PRIVATE_ACCESS = 'private'
+
     def initialize(config)
+      config[:acl] ||= AWS_PUBLIC_ACCESS
```

Comment:
modifying the passed in object is kinda meh ... ideally do `@config = {acl: AWS_PUBLIC_ACCESS}.merge(config)` ... or `dup` if before modifying